deploy/webhooks/nodemaintenance.webhook.yaml

```diff
@@ -28,4 +28,4 @@ webhooks:
       - v1beta1
       # - v1 enable this as soon as we vendor a version of controller runtime which understands it
     sideEffects: None
-    timeoutSeconds: 5
+    timeoutSeconds: 15
```

Comment:
You hit some timeouts?

Reply:
yes, but tbh they might have been caused by misconfiguration of RBAC in an earlier stage. However, the default 30 seconds, so with 15 we are still really fast ;)